src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * AddToCart constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
```

Comment:
You are mixing SKU with units.
Buying 2x1.5l of juice is not the same as 2l+1l, they have different SKUs and should be threated differently even if the sum in units is same.